### PR TITLE
Nehatelhan add page

### PR DESCRIPTION
**Pull request by NehaTelhan**

### CODE REVIEW

File: _pages/NehaTelhan.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: NehaTelhan
+pageURL: //NehaTelhan.github.io/
```
Comment (hectorsector) [inline _pages/NehaTelhan.yaml]: @NehaTelhan just remove the `//` and `/` and you should be all set

File: _pages/nt7ab.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: nt7ab
+pageUrl: https://nt7ab.github.io/
+timestamp: 2017-01-18
+---
+
```
Comment (hectorsector) [inline _pages/nt7ab.yaml]: @NehaTelhan since you've introduced the other file, you can remove this file altogether

**Comment by crichID** [comment]
@NehaTelhan I went ahead and fixed the syntax issues in https://github.com/githubschool/on-demand-github-pages/pull/248/commits/afa50f6affe27fecce3e310c429c78b5f36dfba8. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Comment by brianamarie** [comment]
Hi @NehaTelhan! I've made some changes. Since it's been sitting for a while and the PR is ready, I'm going to go ahead and merge. Nice work!

**Pull request merged**